Rule/Rule/Base.ts

```diff
@@ -13,14 +13,20 @@ export interface Base {
 export namespace Base {
 	export const kinds = ["authorization", "outbound", "inbound"] as const
 	export type Kind = typeof kinds[number]
+	export namespace Kind {
+		export const type = isly.string<Kind>(kinds)
+	}
 	export const categories = ["fincrime", "product", "customer"] as const
 	export type Category = typeof categories[number]
+	export namespace Category {
+		export const type = isly.string<Category>(categories)
+	}
 	export const type = isly.object<Base>({
 		code: isly.string(new RegExp(/^[a-z0-9\-_]+$/)),
```

Comment:
`new RegExp` is unnecessary, since the argument is already a regex pattern, so we can probably clean this up.

Reply:
\+